components/inputmethod/ibus-hangul/history

```diff
@@ -0,0 +1 @@
+system/input-method/library/ibus/hangul@1.5.5,5.11-2025.0.0.0 system/input-method/ibus/hangul
```

Comment:
Please replace this line with
system/input-method/library/ibus/hangul@1.5.5,5.11-2025.0.0.1

The fmri in a history file needs to be the latest. To achieve this the COMPONENT_REVISION has to be incremented. The optional 2nd column is only needed if you want to make sure that existing installations of the old fmri will be replaced with the new one in the 2nd column. This is most probably not needed here because of the file collision in the package.